clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1256,6 +1256,15 @@ public static class ConfigKey {
         public final boolean internalConfig;
         public final String alternativeString;
 
+        public ConfigKey(String name, Type type, Object defaultValue, Validator validator,
```

Comment:
Could you please add comments to say that we keep this method due to backward compatibility?

Reply:
Done.